LambdaEngine/Source/Rendering/LightRenderer.cpp

```diff
@@ -360,7 +360,7 @@ namespace LambdaEngine
 
 		DescriptorHeapDesc descriptorHeapDesc = { };
 		descriptorHeapDesc.DebugName = "Physics Renderer Descriptor Heap";
-		descriptorHeapDesc.DescriptorSetCount = 256;
+		descriptorHeapDesc.DescriptorSetCount = 456;
```

Comment:
LightRenderer now uses descriptor sets which are provided through UpdateDrawArgs(), this in turn avoids the descriptorHeap in lightrenderer to run out of memory therefore this can be changed back to 256